codersdk/healthsdk/health.go

```diff
@@ -1,20 +1,29 @@
-package codersdk
+package healthsdk
```

Comment:
Can we call this `healthsdk.go` instead? Just helps mentally a bit since we do this with other packages.